src/main/java/me/paypur/tconjei/jei/AbstractToolStatsCategory.java

```diff
@@ -87,8 +87,17 @@ protected void drawShadow(PoseStack stack, String string, int x, float lineNumbe
     }
 
     protected void drawStatsShadow(PoseStack stack, Component component, float lineNumber, int color) {
-        String[] strings = component.getString().split(":");
-        strings[0] += ":";
+        String stringraw = component.getString();
+        String[] strings;
+        if (stringraw.contains(":")) {
+            strings = stringraw.split(":");
+            strings[0] += ":";
+        } else if (stringraw.contains("：")) {
+            strings = stringraw.split("：");
+            strings[0] += "：";
+        } else {
+            throw new RuntimeException("Unsupported language");
```

Comment:
I don't think throwing an exception is the best way to handle this because then the JEI category will simply not work. Drawing the whole string with the default text color probably is better.